CryptoAnalysis/src/main/java/crypto/constraints/ConstraintSolver.java

```diff
@@ -511,7 +516,7 @@ else if(value.getKey().equals("false"))
 				}
 				catch (NumberFormatException ex1) {
 					// If that does not work either, I'm out of ideas ...
-					throw new RuntimeException();
```

Comment:
Throwing the exception here seems to be fine, but i'm not sure. 
`valuesInt` is an empty map in this case and i'm not sure if it's allowed to be. If so there probably is no need to throw the exception and thus terminate the whole operation.